### DISCUSSION TITLE
StreamDeck+ support?

### DISCUSSION THREAD

**Discussion by LogicalUnit** [StreamController/StreamController#115] [Q&A]
I have a StreamDeck+ device with touch panel and rotary buttons. Even if those other features aren't supported yet, it would be great if I could use the 8 normal StreamDeck buttons in the usual fashion. Is this supported?

**Comment by LogicalUnit** [comment]
Just installed the flatpak and the eight buttons do work as expected :)

**Reply by v4u6h4n** [reply]
Apparently dials and touch panel are also [in the works](https://github.com/StreamController/StreamController/issues/43#issuecomment-2127768985) now.

**Comment by Cfomodz** [comment]
**Yes.** The StreamDeck+ is fully supported for standard button functionality.

### Working Features
- ✅ All 8 standard buttons work perfectly
- ✅ Full action and plugin support
- ✅ Page switching capabilities

### In Development
- 🔄 Rotary dials (active development)
- 🔄 Touch panel (swipe gestures fixed in PR #496, November 2025)
